main.py

```diff
@@ -52,7 +52,18 @@ async def print_halolink_schema(self):
 
     def print_redcap_data_biopsy_id(self, biopsy_id: str):
         print(self.redcap_connection.get_by_biopsy_id(biopsy_id))
-
+    
+    async def verify_slide_counts(self, biopsy_id: str):

```

Comment:
Please split this function into two: create a new file+class in the "services" dir called "pipeline_service" and create a function to return true or false based on this result of this. In this main file, read the result and print out the "slides match/do not match" stuff. Thanks!